cmd/updater/update.sh

```diff
@@ -93,6 +100,9 @@ while [ "$1" != "" ]; do
             shift
             TOOLS_OUTPUT_DIR=$1
             ;;
+        -z)
+            TESTING=true
```

Comment:
This is a released file, so i'm not sure if we want test mode, but it would make sense to have a dryrun mode.

Reply:
I'll make the change.

Reply:
Fixed